src/Competencies.js

```diff
@@ -0,0 +1,323 @@
+import React, { Component } from 'react';
+import { Layout, Modal, Switch, Icon, Button, Row, Col, Table, Card, message } from 'antd';
+import { getData, update, getLastIndex } from './firebase';
+import _ from 'lodash';
+import Loading from './Loading';
+import CompetenciesCreateForm from './CompetenciesCreateForm'
+import { Link } from 'react-router-component';
+const { Header } = Layout;
+import logo from './logo.png';
+
+class Competencies extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      visible: false,
+      showEditPopup: false,
+      optionActivated: false,
+      showEditPopupKmsOptional: false,
+      optionActivatedKmsOptional: false,
+      loading: true,
+      loadingActivated: true,
+    };
+  }
+
+  handleSaveActivated = (e) => {
+    let dataUpdate = {
+      "activated": this.state.optionActivated,
+      "name": this.state.competencyName
+    }
+
+    if (_.isEqual(this.state.selectOption, 'Kms_core')) {
+      _.update(this.state.competenciesKMSCore[this.state.keyUpdate], 'activated', () => { return this.state.optionActivated })
```

Comment:
Arrow function doesn't need return. We can go with () =? this.state.optionActivated